src/public/components/pretty-stringify.js

```diff
@@ -0,0 +1,7 @@
+import K, * as U from "karet.util"
+import React     from "karet"
+
+export const PrettyStringify = ( { space, obj } ) =>
+  <pre className="pretty-stringify">
+    { U.lift( x => JSON.stringify( x, null, Number( space ) ) )( obj ) }
+  </pre>
```

Comment:
Agreed. I'll add your suggested changes and default `space` to 2. Should I force the default if `space` `isNaN`?